dc_utils/settings/pipeline.py

```diff
@@ -2,7 +2,10 @@
 
 root_path = os.path.dirname(os.path.abspath(__file__))
 
-STATICFILES_STORAGE = "dc_utils.storages.StaticStorage"
+STORAGES = {
+    "default": {"BACKEND": "django.core.files.storage.FileSystemStorage"},
+    "staticfiles": {"BACKEND": "dc_utils.storages.StaticStorage"},
+}
```

Comment:
Same here. `STORAGES` is available from 4.2 onwards
https://docs.djangoproject.com/en/4.2/ref/settings/#storages
so this is compatible with django>=4.2